playbooks/pre_cloud.yaml

```diff
@@ -19,13 +19,20 @@
         mode: "0600"
       no_log: true
 
-    - name: Create ssh key file
+    - name: Create signing key file
```

Comment:
you deal with the secret here, no_log is a must since you get secret from variable which might be exposed

Reply:
This has nothing to do with this PR, but I can do no_log

Reply:
![Screenshot from 2024-09-06 11-29-30](https://github.com/user-attachments/assets/f0e56fda-e274-4112-9e4a-b0a2007cc8c6)
Zuul seems to be smart enough to hide the secret. It says "content: NULL" in the log.
Still, no_log shouldn't hurt. I can do that

Reply:
thanks, you should not ever trust any "secret detection" when you know you deal with sensitive data